### DISCUSSION TITLE
For two cameras, Can i set the rtsp stream with the same ip?

### DISCUSSION THREAD

**Discussion by Gogh-123** [bluenviron/mediamtx#2225] [Questions]
When I push video streams from two cameras at the same time on one processor. Can i set the rtsp://172.16.30.108:8554/mystream0 and rtsp://172.16.30.108:8554/mystream1 for two cameras?

**Comment by aler9** [comment]
The question is not clear, please describe in a detailed way what you're trying to accomplish.

**Reply by Gogh-123** [reply]
My requirement is to connect two gmsl cameras on an industrial computer, I run the mediamtx server, and then use opencv+gstreamer to push the data of both cameras to the rtsp server. Camera 1 push flow address is set to the RTSP: / / 172.16.30.108:8554 / mystream0, camera 2 push flow address is set to the RTSP: / / 172.16.30.108:8554 / mystream1, Therefore, only suffixes are used to distinguish the rtsp stream data of two different cameras. However, when the actual pull stream, only the rtsp of camera 1 has data, can not access the rtsp stream data of camera 2, how can I set up the use of mediamtx, in order to support the processing of multiple RTSP streams at the same time, and can distinguish different streams by different suffixes.

**Reply by aler9** [reply]
The server supports publishing multiple streams each in a separate path, in the order of thousands. No additional configuration is required.

If the second stream  is not available, it means that your OpenCV script is not sending data.

If you still have troubles, provide server logs.

**Reply by Gogh-123** [reply]
thanks, i will try.

[a comment by an automated account is not shown]
